src/assets/styles/mixins/shared-checkbox-styles.scss

```diff
@@ -0,0 +1,38 @@
+// Mixin for shared input styles
+
+@mixin shared-checkbox-styles {
+  .ao-checkbox {
```

Comment:
there is a pr up to change the classname from ao-checkbox to ao-radio for radio buttons,  so we need to be wary of that change

Reply:
https://github.com/AmpleOrganics/Blaze.vue/pull/302

Reply:
cool, see my new commit